main.js

```diff
@@ -0,0 +1,149 @@
+    title: 'Berkman Professor of Entreprenuerial Legal Studies at Havard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal book the wealth of networks in 2006',
+  },
+  {
+    id: 'speaker1',
+    img: {
+      src: './images/2c.PNG', alt: 'Picture of a professor',
+    },
+    name: 'Lila Treklty',
+    title: 'Berkman Professor of Entreprenuerial Legal Studies at Havard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal book the wealth of networks in 2006',
+  },
+  {
+    id: 'speaker1',
+    img: {
+      src: './images/3c.PNG', alt: 'Picture of a professor',
+    },
+    name: 'Kilnam Chon',
+    title: 'Berkman Professor of Entreprenuerial Legal Studies at Havard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal book the wealth of networks in 2006',
+  },
+  {
+    id: 'speaker1',
+    img: {
+      src: './images/4c.PNG', alt: 'Picture of a professor',
+    },
+    name: 'Julia Leda',
+    title: 'Berkman Professor of Entreprenuerial Legal Studies at Havard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal book the wealth of networks in 2006',
+  },
+  {
+    id: 'speaker1',
+    img: {
+      src: './images/5c.PNG', alt: 'Picture of a professor',
+    },
+    name: 'Ryan Merkley',
+    title: 'Berkman Professor of Entreprenuerial Legal Studies at Havard Law School',
+    description: 'Benkler studies commons-based peer production, and published his seminal book the wealth of networks in 2006',
+  },
+];
+function speakerCard(speakerObj) {
+  const {
+    id,
+    img,
+    name,
+    title,
+    description,
+  } = speakerObj;
+  const speakerDomNode = `
+ <div id="${id}" class="speaker">
+ <div class="imge">
+   <img id="imge" src="${img.src}" alt="${img.alt}">
+ </div>
+ <div class="txt">
+  <h4 class="names">${name}</h4>
+  <p class="title">${title}</p>
+  <span class="bard"></span>
+  <p class="des">${description}</p>
+ </div>
+</div>
+`;
+  return speakerDomNode;
+}
+const speakerSection = document.querySelector('.speaker-container');
+speakers.forEach((speaker) => {
+  speakerSection.insertAdjacentHTML('afterbegin', speakerCard(speaker));
+});
+
+/* const moreBtn = document.querySelector('.morebtn');
+const speakerSection = document.querySelector('.speaker-container');
+
+let start = 0;
+let end = 2;
+
+window.onload = () => {
+  const arr = speakers.slice(start, end);
+  arr.forEach((speaker) => {
+    speakerSection.insertAdjacentHTML('afterbegin', speakerCard(speaker));
+  });
+};
+
+moreBtn.addEventListener('click', () => {
+  start = end;
+  end += 2;
+  const arr = speakers.slice(start, end);
+  if (arr.length <= 0) {
+    moreBtn.setAttribute('disabled', true);
+    return;
+  }
+  arr.forEach((speaker) => {
+    speakerSection.insertAdjacentHTML('afterbegin', speakerCard(speaker));
+  });
+}); */
```

Comment:
- [ ] Please remove this commented block of code in order to keep your code clean and make it easy to read! 💯 🤓